lib/example.rb

```diff
@@ -0,0 +1,21 @@
+# An example file to demonstrate how to use the BowlingScore class
+
+require_relative 'bowling_score'
+
+# Less than a full game
+example_1 = BowlingScore.new('XXX')
+puts example_1.total_score
+# => 60
+
+example_2 = BowlingScore.new('95/X')
+puts example_2.total_score
```

Comment:
Should this not be 44?

9+5+10 gives 24 - a strike on your next shot would add 20 to the score.